packages/@react-spectrum/s2/src/style-utils.ts

```diff
@@ -83,14 +83,7 @@ export const fieldLabel = () => ({
   color: {
     default: 'neutral-subdued',
     isDisabled: 'disabled',
-    staticColor: {
-      white: {
-        default: 'transparent-white-700'
-      },
-      black: {
-        default: 'transparent-black-900'
-      }
-    },
+    staticColor: staticColor('transparent-overlay-1000'),
```

Comment:
Design updated the static color labels of progress bar and meter to be fully opaque